src/Excel.php

```diff
@@ -92,7 +92,10 @@ public function download($export, string $fileName, string $writerType = null, a
      */
     public function store($export, string $filePath, string $diskName = null, string $writerType = null, $diskOptions = [])
     {
-        if ($export instanceof ShouldQueue) {
+        if (
+            $export instanceof ShouldQueue ||
```

Comment:
I don't think this will work as expected? In case there's a ShouldQueue interface it will always queue and ignore the method.

Reply:
For me, if something implement shouldQueue it's automatically enqueued. That's why I set up this condition like this (I want to follow the logic of Laravel). The method has an effect only if we assume it's not queued by default. I'm willing to change the condition if you think it's confusing.

Reply:
I believe Laravel does the same: needs the interface and then it checks the method if exists.